spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
Sorry, I goofed up. I had started a what I thought was a different solution (changing the external encoding in `spec/support/file_helper.rb`).

I get the error you reported `invalid byte sequence in US-ASCII` without this fix but not with it.